doc/roadmap.rst

```diff
@@ -124,28 +104,17 @@ the document up to date as we work on these issues.
 #. Make it easier for external users to write Scikit-learn-compatible
    components
 
-   * More flexible estimator checks that do not select by estimator name
-     |ss| :issue:`6599` |se| :issue:`6715`
-   * Example of how to develop an estimator or a meta-estimator,
-     |ss| :issue:`14582` |se|
    * More self-sufficient running of scikit-learn-contrib or a similar resource
 
-#. Support resampling and sample reduction
-
-   * Allow subsampling of majority classes (in a pipeline?) :issue:`3855`
-   * Implement random forests with resampling :issue:`13227`
```

Comment:
Both of these are in imbalanced-learn but not in scikit-learn.

Reply:
The `BalancedRandomForest` PR has been closed: https://github.com/scikit-learn/scikit-learn/pull/13227 and this is the reason that I removed it.

I should put back the subsampling because it might require some discussion to know if this is still in the roadmap or priorities.